commons-app/src/main/java/cz/cuni/mff/xrg/odcs/commons/app/conf/AppConfig.java

```diff
@@ -152,4 +175,57 @@ public AppConfig getSubConfiguration(ConfigProperty namespace) {
     public Properties getProperties() {
         return (Properties) prop.clone();
     }
+
+    private static void postprocess(Properties properties) {
```

Comment:
Yes, but this method creates new instance of AppConfig, so it could be called as: config.postprocess();